.editorconfig

```diff
@@ -1,7 +0,0 @@
-[*]
-end_of_line = lf
-insert_final_newline = true
-indent_size = 2
-indent_style = space
-max_line_length = 120
-quote_type = single
```

Comment:
To confirm, are all these properties the default in prettier? Or do we need to override any of them in prettier to maintain consistent behavior?

Reply:
some of them are default, and most of them we set in `prettier-config.js` and `pretter-scss-config.js`.